101-vm-with-data-management-gateway/azuredeploy.json

```diff
@@ -0,0 +1,297 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "existingDataFactoryName": {
+      "type":"string",
+      "metadata": {
+        "description": "your existing data factory name"
+      }
+    },
+    "GatewayName": {
+      "type": "string",
+      "metadata": {
+        "description": "Gateway name must be unique in subscription"
+      }
+    },
+    "virtualMachineName": {
+      "type": "string",
+      "maxLength": 15,
+      "minLength": 3,
+      "metadata": {
+        "description": "Input the virtual machine name"
+      }
+    },        
+    "Size": {
```

Comment:
The vm sizes in the drop list is available for all regions. Since the capacity of our gateway is determined by the machine size, so we want to provide a option for customers to choose the size.